server/plugin/webhook.go

```diff
@@ -385,18 +390,22 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 
 	for _, sub := range subs {
 		if !sub.Pulls() && !sub.PullsMerged() && !sub.PullsCreated() {
+			p.API.LogDebug("Rejecting event for current channel as subscription is not present for PR events.", "Channel", sub.ChannelID)
 			continue
 		}
 
 		if sub.PullsMerged() && action != actionClosed {
+			p.API.LogDebug("Rejecting event for current channel as event is for PR closed but the subscription is present for event PR merged.", "Channel", sub.ChannelID)
 			continue
 		}
 
 		if sub.PullsCreated() && action != actionOpened {
+			p.API.LogDebug("Rejecting event for current channel as event is for PR opened but the subscription is present for event PR created.", "Channel", sub.ChannelID)
 			continue
 		}
 
 		if p.excludeConfigOrgMember(event.GetSender(), sub) {
+			p.API.LogDebug("Rejecting event for current channel due to exclude config org member condition.", "Channel", sub.ChannelID)
```

Comment:
Probably want to log at all the return statements below as well, with `p.logForDocsRepoDebugging`